modules/default/newsfeed/newsfeedfetcher.js

```diff
@@ -20,7 +20,7 @@ const stream = require("stream");
  * @param {boolean} logFeedWarnings If true log warnings when there is an error parsing a news article.
  * @class
```

Comment:
Missing JSDoc @param "useCorsProxy" declaration

Reply:
fixed ... but should these warning not treated as errors? Or will you check every test manually in the future?

Reply:
na, wont check manually since the warning shows up here in the changes ... erroring it might trigger the old ones, I fear...